react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java

```diff
@@ -138,6 +139,7 @@ public void onSelectionChanged(int selStart, int selEnd) {
                 ReactAztecText.this.propagateSelectionChanges(selStart, selEnd);
             }
         });
+        this.setImeOptions(getImeOptions() | EditorInfo.IME_FLAG_NO_EXTRACT_UI);
```

Comment:
This is a great improvement! Read through IME options [here](https://developer.android.com/reference/android/view/inputmethod/EditorInfo#IME_FLAG_NO_EXTRACT_UI) and agree IME_FLAG_NO_EXTRACT_UI looks like the right choice.